.eslintrc.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "next/core-web-vitals"
+  "extends": "next/core-web-vitals",
+  "rules": {
+    // ESLint is throwing errors with using <a> tags for routing to the login page, this removes the error
+    "@next/next/no-html-link-for-pages": "off"
+  }
```

Comment:
I can copy and paste what we use at my work if you want for Eslint, it's noice

Reply:
Sounds good!